Source/Core/Core/API/Events.h

```diff
@@ -184,8 +189,16 @@ class GenericEventHub final
 };
 
 // all existing events need to be listed here, otherwise there will be spooky templating errors
-using EventHub = GenericEventHub<Events::FrameAdvance, Events::FrameDrawn, Events::SetInterrupt, Events::ClearInterrupt,
-                                 Events::MemoryBreakpoint, Events::SaveStateLoad, Events::CodeBreakpoint>;
+using EventHub = GenericEventHub<
+  Events::FrameAdvance,
+  Events::FrameDrawn,
+  Events::SetInterrupt,
+  Events::ClearInterrupt,
+  Events::MemoryBreakpoint,
+  Events::SaveStateSave,
+  Events::SaveStateLoad,
+  Events::CodeBreakpoint
```

Comment:
It would probably be nicer if `CodeBreakpoint` was next to `MemoryBreakpoint`, both here and other relevant places.

Reply:
Reorganizing, and making it consistent across all locations.